lib/actions/projectOverviewResponse.js

```diff
@@ -0,0 +1,12 @@
+class ProjectOverviewResponse {
+  constructor(links) {
+    this.links = links;
+  }
+
+  sendToBot(bot, message) {
+    const listLinks = this.links.map(({ link, label }, i) => `${i + 1}. ${label} ${link}`).join('\n');
```

Comment:
One of the notes in [PBOT-93](https://sparkbox.atlassian.net/browse/PBOT-93) is
> Consider using the Slack Message Builder to practice and share these options with the client or a designer for feedback.

Have we done this?

Reply:
Nope, will do.